src/goInstallTools.ts

```diff
@@ -115,9 +115,20 @@ function installTools(goVersion: SemVersion, missing?: string[]) {
 
 	outputChannel.appendLine(''); // Blank line for spacing.
 
+	// http.proxy setting takes precedence over environment variables
+	let httpProxy = vscode.workspace.getConfiguration('http').get('proxy');
+	let env = process.env;
+	if (httpProxy) {
+		env = Object.assign({}, process.env, {
+			http_proxy: httpProxy,
```

Comment:
Yes I believe there are.

In case `go get` are calling some external tools such as `wget` to download third party resources or other tools like `wget` which read lowercase setting from environment variables:  https://www.gnu.org/software/wget/manual/wget.html#Proxies

So I think setting both can be safer.